examples/blockchain/main.rs

```diff
@@ -0,0 +1,124 @@
+use futures::{channel::oneshot, StreamExt};
+use log::info;
+
+use aleph_bft::NodeIndex;
+use chain::{gen_chain_config, run_blockchain, DataIO};
+use chrono::Local;
+use crypto::KeyBox;
+use network::{Network, Spawner};
+use std::{io::Write, time};
+
+mod chain;
+mod crypto;
+mod network;
+
+const DATA_SIZE: usize = 1000000;
```

Comment:
If this is supposed to represent a specific number of "transactions" maybe write it out as such, `const TRANSACTION_SIZE...` etc.

Reply:
Good idea, done.